Gemfile

```diff
@@ -6,10 +6,13 @@ ruby RUBY_VERSION
 
 gemspec
 
-DECIDIM_VERSION = "~> 0.26.5"
+DECIDIM_VERSION = "~> 0.27.2"
 
 gem "decidim", DECIDIM_VERSION
 
+# temporal solution while gems embrace new psych 4 (the default in Ruby 3.1) behavior.
+gem "psych", "< 4"
```

Comment:
Can you run `bundle install` to install this gem?

Reply:
I have run it before but it turned out that Gemfile.lock was in .gitignore. I have fixed that so Gemfile.lock is in the repo again